yarn.lock

```diff
@@ -6601,6 +6616,11 @@ optionator@^0.9.1, optionator@^0.9.3:
     prelude-ls "^1.2.1"
     type-check "^0.4.0"
 
+"opts@>= 1.2.0":
```

Comment:
What is this package?

Reply:
didnt added it externally may be it was added with the live reload.